testnet/stacks-node/src/config.rs

```diff
@@ -211,7 +213,7 @@ mod tests {
             ConfigFile::from_str(&format!(
                 r#"
                     [[burnchain.affirmation_overrides]]
-                    reward_cycle = 1
```

Comment:
is this meant to only apply ovverrides for the known cycles (starting at 413), vs the existing method from cycle 1?